src/main/sd/sd.h

```diff
@@ -0,0 +1,58 @@
+#ifndef MEXT2_SD_H
+#define MEXT2_SD_H
+#include <stdint.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+
+#define __packed __attribute__((packed))
+
+#define OCR_REGISTER_LENGTH 4
+#define COMMAND_ARGUMENT_SIZE 4
+#define N_CYCLES_TIMEOUT        (uint16_t) 512
+
+typedef enum mext2_response_type
+{
+    MEXT2_R1,
+    MEXT2_R1b,
+    MEXT2_R2,
+    MEXT2_R3,
+    MEXT2_R7
+} mext2_response_type;
+
+struct mext2_command
+{
+	uint8_t index;
```

Comment:
Let's be consistent in our indentation method. I propose as we started 4 spaces per indent instead of tabs

Reply:
corrected